packages/components/src/Flex/types.ts

```diff
@@ -0,0 +1,13 @@
+export type ColumnKeys = "shrink" | "grow" | "auto";
```

Comment:
@darryltec Are you okay with surfacing `auto` as well? Since templates will convert this to a grid container, sometimes these will be necessary to accomplish a layout. 

For ref, grow is `1fr`, shrink is `min-content` and auto is `auto` (obvi).

Reply:
use `max-content` for shrink so it doesn't squish the elements into oblivion

1fr and auto are closely the same so either grow is 1fr or auto

Reply:
Would we never want an element to shrink down to its `min-content` to fit in a tight spot?

![image](https://github.com/GetJobber/atlantis/assets/10996915/401b3bf9-46a4-4ec5-a976-54ef497fa42f)

Reply:
that screenshot is max-content, no?

https://codepen.io/notdarryltec/pen/GRPYMda

pretty rare for us to actually allow this 
<img width="466" alt="image" src="https://github.com/GetJobber/atlantis/assets/15986172/1c8877ee-6825-4fcc-8298-5097bec64610">

Reply:
i am cool with that